other/quiz/quiz.js

```diff
@@ -115,6 +115,11 @@ const newQuiz = async (guild, reoccur = false) => {
 
   const incorrectFilter = quiz_channel.createMessageCollector({ filter: (m) => !filter(m), time: time_limit});
   incorrectFilter.on('collect', async m => {
+    // stop doing emoji when we're already done pls thx you
+    if (winners.has(user.id) || m.createdTimestamp - finished > ANSWER_TIME_LIMIT) {
```

Comment:
I think this just won't run, as finished is `0` beforehand, need to wait until finished is defined?
```suggestion
    if (finished && (winners.has(user.id) || m.createdTimestamp - finished > ANSWER_TIME_LIMIT)) {
```

Reply:
did I update while you were reviewing?

Reply:
Yeah must have, koz when I reloaded the changes where already fixed lol